.craft.ini

```diff
@@ -115,7 +115,7 @@ Packager/CacheVersion = ${GeneralSettings:Packager/CacheVersion}/squish
 [linux-gcc-x86_64-squish-BlueprintSettings]
 libs/dbus.ignored = False
 libs/icu.ignored = False
-libs/qt6.version = 6.6.2
+libs/qt6.version = 6.7.3
```

Comment:
I tried to build client with craft from this branch. but still the craft was searching for qt-6.8.0.
I found that [`.craft.shelf`](
https://github.com/owncloud/client/blob/master/.craft.shelf) controls the qt version to be used. Qt version was pinned in https://github.com/owncloud/client/pull/11950

Reply:
Can you test it again but remove .craft.shelf? Once we change the qt versions we anyhow need to recreate the shelf.

Reply:
`.craft.shelf` is needed else the command fails with 404 error. But upon removing pinned qt versions from `.craft.shelf`, `ownbuild.py` downloads the correct qt version - `6.7.3`

Reply:
feel free to commit this change to this pull request - thx

Reply:
I will retrigger building the squish docker image ....

Reply:
I will push the changes. But the overall craft build failed.

```
/usr/bin/ld: libelf_pic.a(elf_error.os): in function `elf_errmsg':
/mnt/workspace/owncloud/ownbuild/check-craft-build-squish/linux-gcc-x86_64-squish/build/libs/elfutils/work/elfutils-0.188/libelf/elf_error.c:353: undefined reference to `libintl_dgettext'
/usr/bin/ld: /mnt/workspace/owncloud/ownbuild/check-craft-build-squish/linux-gcc-x86_64-squish/build/libs/elfutils/work/elfutils-0.188/libelf/elf_error.c:350: undefined reference to `libintl_dgettext'
collect2: error: ld returned 1 exit status
make[2]: *** [Makefile:1347: libelf.so] Error 1
make[1]: *** [Makefile:533: all-recursive] Error 1
make: *** [Makefile:449: all] Error 2
Command ['/usr/bin/make', '-j', '20'] failed with exit code 2
Action: make for libs/elfutils:0.188 FAILED
*** Craft all failed: libs/elfutils after 10s ***
fatal error: package libs/elfutils all failed
Craft stopped with out completing ['libs/elfutils', 'dev-utils/breakpad', 'owncloud/libcrashreporter-qt', 'libs/gpg-error', 'libs/gcrypt', 'libs/libsecret', 'qt-libs/qtkeychain']
Command /usr/bin/python3.10 -X utf8 -u /mnt/workspace/owncloud/ownbuild/check-craft-build-squish/linux-gcc-x86_64-squish/craft/bin/craft.py --unshelve /mnt/workspace/owncloud/ownbuild/check-craft-build-squish/.craft.shelf failed with exit code: 1
Failed to setup craft
```
we are missing `qt-libs` for squish cache:
There should be [qt-libs/qtkeychain/](https://download.owncloud.com/desktop/craft/cache/6.0/Qt_6.7.0_0/squish/linux/gcc/x86_64/RelWithDebInfo/qt-libs/qtkeychain/) in here as well: https://download.owncloud.com/desktop/craft/cache/6.0/Qt_6.8.0_0/squish/linux/gcc/x86_64/RelWithDebInfo/

Reply:
Can we mirror this path for squish target (except for qt6 libs)? Some libraries are missing in `/Qt_6.8.0_0/squish`
https://download.owncloud.com/desktop/craft/cache/6.0/Qt_6.7.3_0/squish/linux/gcc/x86_64/RelWithDebInfo/
mirror to
https://download.owncloud.com/desktop/craft/cache/6.0/Qt_6.8.0_0/squish/linux/gcc/x86_64/RelWithDebInfo/

Reply:
THe cache looks complete now. https://download.owncloud.com/desktop/craft/cache/6.0/Qt_6.8.0_0/squish/linux/gcc/x86_64/RelWithDebInfo

Reply:
I can use craft and build client on this branch. so the cache is good now.